CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+
+cmake_minimum_required (VERSION 2.6)
+project (LIBUAST C CXX)
+
+set (CMAKE_CXX_STANDARD 11)
+
+find_package(ZLIB REQUIRED)
```

Comment:
Will try, but when building the executable, you have to link it with all the static libraries, you can't just link with libxml2.a, since it does not include the other libraries.

If I don't explicitly link with ZLIB, etc, it does not compile.